.github/workflows/pull_request.yml

```diff
@@ -36,18 +36,8 @@ jobs:
     - run: bundle exec rubocop
     - run: bundle exec rspec
     - run: bundle exec yard stats ./lib/line/bot/v2 --fail-on-warning
-
-  rbs:
-    runs-on: ubuntu-latest
-    permissions:
-      contents: read
-    steps:
-      - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
-      - uses: ruby/setup-ruby@354a1ad156761f5ee2b7b13fa8e09943a5e8d252 # v1.229.0
-        with:
-          ruby-version: '3.2'
-      - run: gem install rbs
-      - run: rbs -I sig validate
```

Comment:
I'll remove `rbs` checks from here after this PR gets approve. 
![image](https://github.com/user-attachments/assets/51c35b5f-6646-4f54-a95e-c2aedede41a0)